### DISCUSSION TITLE
Welcome to ViaForge Discussions!

### DISCUSSION THREAD

**Discussion by florianreuth** [ViaVersion/ViaForge#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by u64lisa** [comment]


**Reply by florianreuth** [reply]
thank you

**Reply by John3214s2** [reply]
Hello, really cool project, I wanted to ask if ViaForge will soon be available in version 1.19.2. Unfortunately, you cannot switch to 1.19.2 with version 1.12.2. It's much better not having to keep restarting Minecraft to get into a different version.

**Reply by florianreuth** [reply]
It's available for 1.19.2

**Reply by StateBros** [reply]
This is cool, i have a 1.21.1 neoforge server but i cant find a viaforge for 1.21.1, does that exist anywhere?

**Reply by HowardZHY** [reply]
> This is cool, i have a 1.21.1 neoforge server but i cant find a viaforge for 1.21.1, does that exist anywhere?

ViaForge is client-side, you may try SpongeNeo with [ViaSponge](https://github.com/ViaVersion/ViaSponge) on Server

**Comment by 187smkz** [comment]
is it possible to use this mod on 1.7.10?

**Reply by HowardZHY** [reply]
No, 1.7.10 Support is not planned

**Reply by florianreuth** [reply]
You can use the ViaProxy for that

**Reply by 187smkz** [reply]
tysm

**Comment by Outfit8TSB** [comment]
is ViaForge going to support NeoForge?

**Reply by florianreuth** [reply]
It's not planned at the moment, ViaProxy can be used for that.

**Reply by Outfit8TSB** [reply]
oh, okay. (in fact, i like viaproxy)

**Comment by Im3sa** [comment]
so when im on 1.8.9 main menu and i click "viaforge" button on the top left it opens a empty screen. i mean it just nothing its empty, so i cant choose a version i wanna use

**Reply by HowardZHY** [reply]
> so when im on 1.8.9 main menu and i click "viaforge" button on the top left it opens a empty screen. i mean it just nothing its empty, so i cant choose a version i wanna use

Make sure you'r ViaForge is updated.

Create an issue with your client logs if you're sure this happens on latest

**Comment by ijustputcodeandwork** [comment]
i have new error

fact: all servers use npc plugins (znpcs, citizens) can't access
1.8.9 to server 1.21

**Reply by HowardZHY** [reply]
Try to use ViaProxy, it can be a ViaRewind issue
